ui/app/src/views/admin/tabs/GlobalSecret.tsx

```diff
@@ -16,7 +16,7 @@ import { useCallback, useEffect, useState } from 'react';
 import { useSnackbar } from '@perses-dev/components';
 import { GlobalSecretResource, Secret } from '@perses-dev/core';
 import { CachedDatasourceAPI, HTTPDatasourceAPI } from '../../../model/datasource-api';
-import { SecretList } from '../../../components/secret/SecretList';
+import { SecretList } from '../../../components/secrets/SecretList';
```

Comment:
We should have all tabs aligned here, it seems we started with singular on the code side, and plural in the tabs title. It started diverging with the addition of roles and rolebindings. IMHO better to keep `secret` here and change `roles` folder to `role` and `rolebindings` to `rolebinding`.

Reply:
Agree with you there is an issue with naming, but components is using plural so there will be a mix too